multimodal/src/autogluon/multimodal/data/process_text.py

```diff
@@ -154,10 +154,12 @@ def __init__(
             self.max_len = self.tokenizer.model_max_length
         else:
             if max_len < self.tokenizer.model_max_length:
-                warnings.warn(
-                    f"provided max length: {max_len} "
-                    f"is smaller than {model.checkpoint_name}'s default: {self.tokenizer.model_max_length}"
-                )
+                # TODO, Consider to fix the logic
+                if self.tokenizer.model_max_length < 10**6:
```

Comment:
Why do we need to add this if condition?

Reply:
In some models (like T5 / deberta-v3), the model_max_length is really large so I added the condition here to avoid popping up too many warning messages.

This is mainly to guard against warnings.